library/core/src/Header.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * OpenEMR <http://open-emr.org>.
+ *
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Core;
+
+use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\Yaml\Exception\ParseException;
+
+/**
+ * Class Header.
+ *
+ * Helper class to generate some `<script>` and `<link>` elements based on a
+ * configuration file. This file would be a good place to include other helpers
+ * for creating a `<head>` element, but for now it sufficently handles the
+ * `includeAsset()`
```

Comment:
Need a good description of options and examples of use since developers will look here. Can basically mimick instructions here:
https://github.com/openemr/openemr/blob/master/library/templates/standard_header_template.php#L6-L26